frontend/src/app/person/person.component.spec.ts

```diff
@@ -42,7 +42,7 @@ describe('PersonComponent', () => {
     abroadFamilySituation: null
   };
 
-  const activatedRoute = {
+  const activatedRoute: any = {
```

Comment:
Out of curiosity, I tried to make it compiles. 
This issue is with `data`, you have to write:

```typescript
  const activatedRoute = {
    parent: {
      snapshot: { data: { person } as Data }
    }
  } as ActivatedRoute;
```
A bit ugly, but works.